src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -12,13 +13,36 @@ public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_SUCCESS_PATIENTS = "Listed all patients.";
+    public static final String MESSAGE_SUCCESS_APPOINTMENTS = "Listed all appointments.";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " patients: List down all patients.\n"
+            + COMMAND_WORD + " appts: List down all appointments.\n";
+
+    private final String type;
+
+    public ListCommand(String type) {
+        this.type = type;
+    }
 
 
     @Override
     public CommandResult execute(Model model) {
-        requireNonNull(model);
-        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-        return new CommandResult(MESSAGE_SUCCESS);
+        if (this.type.equals("patients")) {
+            requireNonNull(model);
+            model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+            return new CommandResult(MESSAGE_SUCCESS_PATIENTS);
+        } else {
+            requireNonNull(model);
+            model.updateFilteredAppointmentList(PREDICATE_SHOW_ALL_APPOINTMENTS);
+            return new CommandResult(MESSAGE_SUCCESS_APPOINTMENTS);
+        }
```

Comment:
```suggestion
        requireNonNull(model);
        if (this.type.equals("patients")) {
            model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
            return new CommandResult(MESSAGE_SUCCESS_PATIENTS);
        } else {
            model.updateFilteredAppointmentList(PREDICATE_SHOW_ALL_APPOINTMENTS);
            return new CommandResult(MESSAGE_SUCCESS_APPOINTMENTS);
        }
```
Perhaps the requireNonNull(model) line could be removed and placed at the top before the if else block? This way we can reduce code duplication.